sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py

```diff
@@ -399,6 +397,9 @@ def upload_blob(  # pylint: disable=too-many-locals
         :param metadata:
             Name-value pairs associated with the blob as metadata.
         :type metadata: dict(str, str)
+        :keyword blob_tags:
+            Name-value pairs associated with the blob as tag.
+        :paramtype blob_tags: dict(str, str)
```

Comment:
Is there a reason this isn't just called "tags"? As we're already on the "blob" client

Reply:
Just the field in generated code is called blob_tags, while we can rename it to tags!